controlplane/kubeadm/internal/workload_cluster_conditions.go

```diff
@@ -334,11 +506,20 @@ func (w *Workload) UpdateStaticPodConditions(ctx context.Context, controlPlane *
 			for _, condition := range allMachinePodConditions {
 				conditions.MarkFalse(machine, condition, controlplanev1.PodFailedReason, clusterv1.ConditionSeverityError, "Missing node")
 			}
+
+			for _, condition := range allMachinePodV1beta2Conditions {
+				v1beta2conditions.Set(machine, metav1.Condition{
+					Type:    condition,
+					Status:  metav1.ConditionUnknown,
```

Comment:
Similar question as above, are we sure about Unknown instead of False? (but seems okay to me, there is a chance that the pod is running even if we can't match providerIDs)

Reply:
With the new condition are using consistently unknown where we cannot determine status of the underlying resource, it seems more correct to me